hnclient-v2/src/main/java/ca/bc/gov/hlth/error/ErrorCodes.java

```diff
@@ -7,7 +7,7 @@
 
 	// TODO: Verify error codes with Patrick
 	SERVER_ERROR(500, "Error connecting to SERVER"), HOST_UNAVAILABLE(401, "Could not connect with the remote host"),
-	REMOTE_TIMEOUT_ERROR(402, "Connection with remote facility timed out"), UNAUTHORIZED(403, "Unauthorized");
+	REMOTE_TIMEOUT_ERROR(402, "Connection with remote facility timed out"), UNAUTHORIZED(404, "Unauthorized");
```

Comment:
We should fix these up I think. This is what we're likely to get back from the hns-esb
**400 Bad Request**
**401 Unauthorized**
**403 Forbidden**
**404 Not Found**
**405 Method Not Allowed**
**408 Request Timeout**
**500 Internal Server Error
502 Bad Gateway
503 Service Unavailable
504 Gateway Timeout**

Reply:
Hi Trevor, Please review the changes.